packages/advanced-logic/src/extensions/payment-network/any-to-eth-proxy.ts

```diff
@@ -0,0 +1,158 @@
+import { ExtensionTypes, RequestLogicTypes } from '@requestnetwork/types';
+import EthereumFeeProxyPaymentNetwork from './ethereum/fee-proxy-contract';
+
+const CURRENT_VERSION = '0.1.0';
+// Default network if the storage data does not give any
+const DEFAULT_NETWORK = 'mainnet';
+
+/**
+ * These currencies are supported by Chainlink for conversion.
+ * Only ETH is supported as accepted token by the payment proxy.
+ */
+const supportedCurrencies: Record<string, Record<RequestLogicTypes.CURRENCY, string[]>> = {
```

Comment:
Shouldn't we refactor this ? I don't see good reasons for AnyToErc20 and AnyToEth to not share the supported currencies.